cellpack/tests/test_representations.py

```diff
@@ -68,3 +71,32 @@ def test_unpacked_sphere_tree():
     assert len(positions) > 0
     assert len(radii) > 0
     assert positions == [[[1, 0, 0]]]
+
+
+def test_set_active():
+    representations = Representations(mesh=test_mesh_obj, atomic=test_pdb_id)
+    representations.set_active("atomic")
+    assert representations.get_active() == "atomic"
+    assert representations.get_active_data() == test_pdb_id
+
+
+def test_get_adjusted_position():
+    starting_position = [1, 1, 1]
+    representations = Representations(
+        mesh=test_mesh_obj, atomic=test_pdb_id_obj_with_offset
+    )
+    # if the representation used has a transform, should move the starting position
+    representations.set_active("atomic")
+    adjusted_position = representations.get_adjusted_position(
+        starting_position, np.eye(4)
+    )
+    for i in range(len(adjusted_position)):
+        assert adjusted_position[i] != starting_position[i]
```

Comment:
This only works if the translation occurs in all three dimensions. Could we modify this?
`adjusted_position[i] == starting_position[i] - test_pdb_id_ob_with_offset["transform"]["translate"][i]`

Reply:
I guess I could also make a flag that just turns true if any of them don't match